.benchmarks/Darwin-CPython-3.11-64bit/0001_e8b0acf18413d142a4778eb63f40c518f32b94bc_20241120_094745_uncommited-changes.json

```diff
@@ -0,0 +1,288 @@
+{
```

Comment:
It does not make sense to keep this file in version control, but it is userful to have it as an artifact in CI to make comparisons, and, eventually, tests to fail under a certain threshold (say, <25%).